src/main/java/org/hawkular/apm/qe/JaegerQEBase.java

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
@kevinearls I do not recommend to create `JaegerQEBase` class. We can update these changes on https://github.com/Hawkular-QE/hawkular-apm-qe/blob/master/src/main/java/org/hawkular/apm/qe/tracer/OpenTracing.java

After some time we will support either `Jaeger` or `Hawkular APM`. Not both.